open-apparel-registry/open-apparel-registry
# Remove Sample Data and Update Registration Text

## Overview

- Significant numbers of users are not deleting the first row of sample
data from the OAR Contributor templates, meaning we end up with
"Example facility" as an entry in the tool. Deletes the first row of
sample data from the OAR contributor templates.
- The text on the "Register" page of the site is confusing, now that
users have to create an account & log in before downloading searches.
Updating this text resolves the issue.

Connects #1267 and #725 

## Demo

<img width="769" alt="Screen Shot 2021-03-23 at 2 25 04 PM" src="https://user-images.githubusercontent.com/21046714/112198798-dd34e380-8be3-11eb-8acf-7e7d3d1b6fa1.png">

## Testing Instructions

* Run `./scripts/server` in vagrant
* Navigate to [Registration](http://localhost:6543/auth/register)
     - [x] Confirm the text says "Thank you for your interest in the OAR. In order to download data from, or contribute to, the OAR please first set up a free account:"
* Sign in as a contributor. Navigate to [Contribute](http://localhost:6543/contribute). Download each of the four list templates. 
    - [x] The templates should contain the headers as previously
    - [x] The templates should not contain example data

## Checklist
